Compiling on RHEL8 beta (gcc 8.2.1) gives a single warning : stringop-truncation

Compiling latest sysstat on RHEL8 beta generates a single warning:

    ...
    gcc -o rd_stats.o -c -g -O2 -Wall -Wstrict-prototypes -pipe -O2 -DSOURCE_SADC  -DSA_DIR=\"/var/tmp/mysar/data\" -DSADC_PATH=\"/var/tmp/mysar/lib/sa/sadc\"  -DHAVE_SYS_SYSMACROS_H -DHAVE_SYS_PARAM_H -DUSE_NLS -DPACKAGE=\"sysstat\" -DLOCALEDIR=\"/var/tmp/mysar/share/locale\" rd_stats.c
    rd_stats.c: In function ‘read_fchost’:
    rd_stats.c:2597:4: warning: ‘strncpy’ output may be truncated copying 16 bytes from a string of length 255 [-Wstringop-truncation]
    strncpy(st_fc_i->fchost_name, drd->d_name, MAX_FCH_LEN);
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    gcc -o count.o -c -g -O2 -Wall -Wstrict-prototypes -pipe -O2 -DSOURCE_SADC  -DSA_DIR=\"/var/tmp/mysar/data\" -DSADC_PATH=\"/var/tmp/mysar/lib/sa/sadc\"  -DHAVE_SYS_SYSMACROS_H -DHAVE_SYS_PARAM_H -DUSE_NLS -DPACKAGE=\"sysstat\" -DLOCALEDIR=\"/var/tmp/mysar/share/locale\" count.c

As I [contributed this bit of code](https://github.com/sysstat/sysstat/blame/5ad51d5936c672c315cb3e3a267a95e66318c47a/rd_stats.c#L2196) to sysstat a few years ago, I ought at least try fixing it. https://github.com/sysstat/sysstat/pull/198

--- a/rd_stats.c
+++ b/rd_stats.c
@@ -2594,7 +2594,7 @@ __nr_t read_fchost(struct stats_fchost *st_fc, __nr_t nr_alloc)
 			st_fc_i->f_txframes = tx_frames;
 			st_fc_i->f_rxwords  = rx_words;
 			st_fc_i->f_txwords  = tx_words;
-			strncpy(st_fc_i->fchost_name, drd->d_name, MAX_FCH_LEN);
+			memcpy(st_fc_i->fchost_name, drd->d_name, MAX_FCH_LEN);
 			st_fc_i->fchost_name[MAX_FCH_LEN - 1] = '\0';
 		}
 	}

